### PR TITLE
Update atlas branding

### LINKED ISSUE #3021: Update Atlas Branding
## Context
* #2959 

## Scope
- Exploring and discussing together that capital "A" alternative version
- Updating components & flows:
   - [Adding Atlas logo as a component here](https://www.figma.com/file/Cc3VDoK6qglJ617ChA2EMr/Foundation?node-id=716%3A70391),
   - [Updating the top navigation component](https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=3173%3A135755),
   - [Updating the Side navigation component](https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=4322%3A173750),
   - [Updating the Sign in flow](https://www.figma.com/file/ymHYOFItrmFaLeBC5Z4tpQ/Sign-in?node-id=0%3A1),
   - [Updating the logo in the embedded variant of the Video player component](https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2090%3A120662).

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1202652214192092) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by drillprop**
Fix #3021

### CODE REVIEW

**Comment by kdembler** [comment]
I think we need to add some padding for `apple-touch-icon`. This is how previous version looks like when added to iOS homescreen vs the new one:
IMG_1071

cc @toiletgranny

File: packages/atlas/src/index.html [hunk 1]
```diff
@@ -4,13 +4,17 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Joystream Atlas" />
```
Comment (kdembler) [inline packages/atlas/src/index.html]: `Watch, publish, sell and interact with videos on Joystream network.`

(let's use same for embedded `index.html`)

**Comment by toiletgranny** [comment]
How about using this image as iOS icon? https://www.figma.com/file/Cc3VDoK6qglJ617ChA2EMr/Foundation?node-id=1315%3A10236

**Comment by kdembler** [comment]
I like it 👍 

@toiletgranny I also noticed that we could add some "site image" that would be displayed for example by social media crawlers:

CleanShot 2022-08-12 at 13 46 53@2x

Could you maybe pick some image like this for us?

**Comment by toiletgranny** [comment]
@kdembler sure, but could I ask you to create an issue for that? 🙏 I'd love to take a little bit of time exploring common practices in approaching OpenGraph images and whether we can provide multiple resolutions for smaller and bigger previews.

**Comment by kdembler** [comment]
@drillprop I think `packages/atlas/src/public/apple-touch-icon.png` could be 180x180 instead, that's the default I see mentioned and the current icon is a tiny bit pixelated

**Review by kdembler — APPROVED** [review]

**Pull request merged**